src/puppetlabs/puppetdb/command.clj

```diff
@@ -123,7 +124,7 @@
 
 ;; ## Command submission
 
-(defn-validated enqueue-raw-command! :- s/Str
+(defn-validated ^:private do-enqueue-raw-command :- s/Str
   "Submits raw-command to the mq-endpoint of mq-connection and returns
```

Comment:
Open to better names - just thought we might want a distinction between these and the new protocol methods.